docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +173,81 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+
+* **Suppressed alerts**: Indicates that the alert was created with alert suppression, and shows how many duplicate alerts were suppressed. This information only appears if alert suppression is enabled for the rule.
+* **Alerts related by source event**: Shows alerts that were created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. 
+* **Cases related to the alert**: Shows the total number and names of cases to which the alert has been added.
+* **Alerts related by session ID**: Shows alerts that generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. 
++
+NOTE: To use this feature, you must enable the **Include session data** setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by process ancestry**: Shows alerts that are related by process events on the same linear branch. When you examine this information further, note that alerts generated from processes on child or related branches are not shown. 
++
+NOTE: To access data about alerts related by process ancestry, you must have a https://www.elastic.co/pricing[Platinum or higher subscription].
+
+[discrete]
+[[prevalence-section]]
+=== Prevalence
+
+The Prevalence sub-section shows how often data from the alert was observed on other host and user events. Prevalence calculations use data from the alert's highlighted fields and examine user and host events that occurred during the last 30 days. Click **Prevalence** to further examine prevalence data in the <<prevalence-sub-tab,Prevalence sub-tab>>. 
+
+[discrete]
+[[prevalence-sub-tab]]
+==== Prevalence sub-tab
+
+The **Prevalence** sub-tab on the left panel shows relationships with other alerts, events, users, and hosts. Update the date time picker to show data from a different time range. 
 
 [role="screenshot"]
-image::images/threat-intel-sub-sec.png[Enriched with Threat Intelligence subsection with matched file hash, 700]
+image::images/prevalence-sub-tab-lp.png[Table within the Prevalence sub-tab]
 
-TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+The following details are provided:
+
+* **Field**: Shows <<investigation-section,highlighted fields>> for the alert.
+* **Value**: Shows values for highlighted fields. 
+* **Alert count**: Shows the total number of alert documents that have identical field values, including the alert you're currently examining. For example, if the `host.name` field alert count is 5, there are five total alerts with the same `host.name` field value.
++
+NOTE: Alert documents contain the {ecs-re}ecs-allowed-values-event-kind.html#ecs-event-kind-signal[`event.kind:signal`] field-value pair.  
```

Comment:
@PhilippeOberti would you mind reviewing this note and the one for the document count column as well - thanks!